pkg/cri/server/status.go

```diff
@@ -67,17 +68,19 @@ func (c *criService) Status(ctx context.Context, r *runtime.StatusRequest) (*run
 		}
 		resp.Info["golang"] = string(versionByt)
 
-		cniConfig, err := json.Marshal(c.netPlugin.GetConfig())
+		cniConfig, err := json.Marshal(netPlugin.GetConfig())
 		if err != nil {
 			log.G(ctx).WithError(err).Errorf("Failed to marshal CNI config %v", err)
 		}
 		resp.Info["cniconfig"] = string(cniConfig)
 
-		lastCNILoadStatus := "OK"
-		if lerr := c.cniNetConfMonitor.lastStatus(); lerr != nil {
-			lastCNILoadStatus = lerr.Error()
+		for name, h := range c.cniNetConfMonitor {
+			lastCNILoadStatus := "OK"
+			if lerr := h.lastStatus(); lerr != nil {
+				lastCNILoadStatus = lerr.Error()
+			}
+			resp.Info[fmt.Sprintf("lastCNILoadStatus.%s", name)] = lastCNILoadStatus
```

Comment:
How does this key end up being consumed? Should `lastCNILoadStatus.default` be returned as `lastCNILoadStatus`?

Reply:
Yep, you are right, I think we should do this.